tests/test_quickbooks_discovery.py

```diff
@@ -159,4 +152,12 @@ def test_run(self):
 
                     custom_field_stream_count += 1
                     
-        self.assertEqual(custom_field_stream_count,6)
\ No newline at end of file
+        self.assertEqual(custom_field_stream_count,6)
+
+        actual_fields = []
+        for md_entry in metadata:
+            if md_entry['breadcrumb'] != []:
+                actual_fields.append(md_entry['breadcrumb'][1])
+
+        # Verify there is no duplicate metadata entries
+        self.assertEqual(len(actual_fields), len(set(actual_fields)), msg = "duplicates in the metadata entries retrieved")
```

Comment:
Break the long line to keep consistency with above code.

Reply:
done